services/interop-proxy/lib/interop_proxy_web/controllers/controller_helpers.ex

```diff
@@ -14,9 +14,14 @@ defmodule InteropProxyWeb.ControllerHelpers do
 
   def send_message(conn, _status_code, {:error, reason}) do
     {code, text} = case reason do
-      %HTTPoison.Error{} -> {503, "Interop server not available\n"}
-      :forbidden         -> {503, "Interop server did not accept cookie\n"}
-      other              -> {500, other}
+      %HTTPoison.Error{} ->
+        {503, "Interop server not available\n"}
+      :forbidden ->
+        {503, "Interop server did not accept cookie\n"}
+      {:message, code, message} when div(code, 100) < 4 ->
```

Comment:
I mean, you could have just said `code < 400`, but okay.

Reply:
Yeah I feel kinda dumb now since I was gonna do something special with 4xx only, but just kept the logic like that, but don't feel like recommitting since it works.